engine/schema/mysql.sql

```diff
@@ -3,41 +3,53 @@
 --
 
 -- record membership in an access collection
-CREATE TABLE `prefix_access_collection_membership` (
-  `user_guid` int(11) NOT NULL,
-  `access_collection_id` int(11) NOT NULL,
-  PRIMARY KEY (`user_guid`,`access_collection_id`)
-) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+-- Access collections were replaced by ACLs
+-- See https://docs.google.com/a/elgg.org/document/d/1R3v_bYno6fw8mV5_GDW93uTI0pvR9S4yQ-BHMKTz1zs/edit#
+-- Each user-created "friend collection" is now an entity pointing to an ACL
 
 -- define an access collection
-CREATE TABLE `prefix_access_collections` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
-  `name` text NOT NULL,
```

Comment:
Currently access collections have a (user defined) name. How will that happen in the future? Please let it be something extendable so it can fix #5225